### PR TITLE
Add new module MRP Repair reorder lines

### PR DESCRIPTION
**Pull request by hurrinico**

### CODE REVIEW

File: mrp_repair_reorder_lines/README.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Repair Lines with sequence number
+=================================
+
+This module was written to allow to manage the repair lines order.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (tafaRU) [inline mrp_repair_reorder_lines/README.rst]: Put this inside https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#configuration section

File: mrp_repair_reorder_lines/README.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/129/8.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (tafaRU) [inline mrp_repair_reorder_lines/README.rst]: Please avoid these lines

File: mrp_repair_reorder_lines/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Repair Lines with sequence number",
+    "summary": '''
+    Provide a new field on the repair line form, allowing to manage the lines
+    order.''',
+    "version": "8.0.1.0.0",
+    "category": "Manufacturing",
+    "website": "https://odoo-community.org/",
```
Comment (tafaRU) [inline mrp_repair_reorder_lines/__openerp__.py]: Please replace it by https://www.agilebg.com

File: mrp_repair_reorder_lines/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "author": "Agile BusinessGroup,"
```
Comment (tafaRU) [inline mrp_repair_reorder_lines/__openerp__.py]: s/BusinessGroup/Business Group

File: mrp_repair_reorder_lines/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "base",
```
Comment (tafaRU) [inline mrp_repair_reorder_lines/__openerp__.py]: Please remove it as redundant

**Review by tafaRU — CHANGES_REQUESTED** [review]
Also please add the OCA icon.
Thank you in advance.

**Review by andhit-r — APPROVED** [review]
LGTM :+1:

**Comment by hurrinico** [comment]
@tafaRU i've done the requested changes

**Comment by eLBati** [comment]
@hurrinico please `commit --amend` to restart travis build

**Comment by hurrinico** [comment]
@eLBati Done 💃

**Comment by gurneyalex** [comment]
@hurrinico I'm ok on the module, but the README needs updating. The usage section especially is not useful. Same thing for the HTML version of the readme.

**Comment by hurrinico** [comment]
@gurneyalex i've chaged the usage section, but i don't understand what i've to change about HTML

**Comment by gurneyalex** [comment]
@hurrinico you included the default template file, which needs filling in. I'd recommend maintaining only a correct README.rst file, otherwise you end up duplicating a lot of information which is against DRY. 

Parts which would need updating are 
- https://github.com/OCA/manufacture/pull/154/files#diff-24a1fa7869f73fa834b176a3c4c0a4b3R16 
- I'm pretty sure the sshot is not what you want https://github.com/OCA/manufacture/pull/154/files#diff-24a1fa7869f73fa834b176a3c4c0a4b3R25
- configuration: https://github.com/OCA/manufacture/pull/154/files#diff-24a1fa7869f73fa834b176a3c4c0a4b3R38
- usage: https://github.com/OCA/manufacture/pull/154/files#diff-24a1fa7869f73fa834b176a3c4c0a4b3R60
- ...

**Comment by hurrinico** [comment]
@gurneyalex i'have made some changes to index.html file

**Comment by eLBati** [comment]
@hurrinico I would remove index and keep README only

**Comment by hurrinico** [comment]
@eLBati done

File: mrp_repair_reorder_lines/static/description/icon.svg [hunk 6]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (eLBati) [inline mrp_repair_reorder_lines/static/description/icon.svg]: @hurrinico if you add icon.png , con't add icon.svg

**Review by eLBati — CHANGES_REQUESTED** [review]
then, :+1:

**Review by tafaRU — CHANGES_REQUESTED** [review]
Please squash 2 commits together for merging.

**Comment by LoisRForgeFlow** [comment]
I tried to merge as it was green, but seems that after rebasing it is not. I will close as this is stale, please  reopen and fix it if you think it is still relevant.

**Pull request closed without merging**